tests/services/login.test.js

```diff
@@ -0,0 +1,29 @@
+import {login} from '../../services/login';
+import {buildResponse} from '../../utils/utils';
+
+
+describe('Login Module', () => {
+
+    test('Should return a 400 error if no user is provided', async () => {
+        const response = await login();
+        expect(response).toEqual(buildResponse(400, {message: "No user provided"}));
+    });
+    
```

Comment:
default password and username can be reused.
or we will have to change them cause the parameter can be confusing and might create bugs later.

Reply:
Fixed